src/member/templates/client/partials/forms/referent_information.html

```diff
@@ -32,6 +32,29 @@ <h4 class="ui dividing header">{% trans 'Referent' %}</h4>
               {{form.lastname}}
           </div>
         </div>
+        <div class="field">
+          <label>Contact information</label>
+          <div class="three fields">
+            <div class="field">
+              <div class="ui left icon input">
+                <i class="at grey icon"></i>
```

Comment:
No need to use to the _grey_ class. Semantic UI will style the icon as part of an input entry.